analysis/descriptives/venn.R

```diff
@@ -250,17 +218,8 @@ df[,setdiff(colnames(df),c("outcome"))] <- lapply(df[,setdiff(colnames(df),c("ou
 
 # Rename columns (output redaction) --------------------------------------------
 
-names(df)[names(df) == "only_snomed"] <- "only_snomed_midpoint6"
-names(df)[names(df) == "only_hes"] <- "only_hes_midpoint6"
-names(df)[names(df) == "only_death"] <- "only_death_midpoint6"
-names(df)[names(df) == "snomed_hes"] <- "snomed_hes_midpoint6"
-names(df)[names(df) == "snomed_death"] <- "snomed_death_midpoint6"
-names(df)[names(df) == "hes_death"] <- "hes_death_midpoint6"
-names(df)[names(df) == "snomed_hes_death"] <- "snomed_hes_death_midpoint6"
-names(df)[names(df) == "total_snomed"] <- "total_snomed_midpoint6"
-names(df)[names(df) == "total_hes"] <- "total_hes_midpoint6"
-names(df)[names(df) == "total_death"] <- "total_death_midpoint6"
-names(df)[names(df) == "total"] <- "total_midpoint6_derived"
+colnames(df)[2:12] <- paste(colnames(df)[2:12], "midpoint6", sep = "_")
```

Comment:
I will keep the old approach.

Reply:
The old approach is inappropriate because it assumes the columns occur in the same order in the real data and we do not know that they do because we cannot see the data.

Reply:
Got it.